src/DevicesPage.tsx

```diff
@@ -381,7 +381,7 @@ export default (): React.ReactElement => {
   const navigate = useNavigate();
   const astarte = useAstarte();
 
-  const pageAlerts = useAlerts();
+  const [pageAlerts] = useAlerts();
```

Comment:
I like how separating state from actions makes it evident that we are defining a variable `pageAlerts` that will never be mutated.
By the way, we might remove this line altogether since we are not showing alerts.